evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -71,6 +74,21 @@ fn sign(private_key: U256, digest: H256, chain_id: U256) -> Result<Bytes, Bytes>
     Ok((sig.v, r_bytes, s_bytes).encode().into())
 }
 
+fn derive_key(mnemonic: &str, path: &str, index: u32) -> Result<Bytes, Bytes> {
+    let derivation_path = format!("{}{}", path, index);
```

Comment:
i'd rather not assume that the path ends in a `/`

Reply:
I originally wanted to do that, but changed it to [fit in line more with ethers-rs](https://github.com/gakonst/ethers-rs/blob/cffcc7bbec2f34bc0b11a522d374b816587070e4/ethers-signers/src/wallet/mnemonic.rs#L114).

With that said, I actually think I made it worse lol.

I'd be fine to add in the `/` for the path so that the user is not expected to. We could also hold the users hand and check for it and only add it if it's not there.

Reply:
In that case, it's a const and not a user input though...

> We could also hold the users hand and check for it and only add it if it's not there.
I like it